cellprofiler/utilities/cpjvm.py

```diff
@@ -67,7 +67,7 @@ def get_patcher_args(class_path):
 
     patchers = filter(
         (lambda x:os.path.split(x)[1].startswith("prokaryote")), class_path)
-    if len(patchers) > 0 and False:
```

Comment:
This was in the code to prevent the patcher from running until Prokaryote-1.0.4 was released with the necessary javaagent stuff. I moved the version # to setup.cfg at about the same time as you released 1.0.4 and I figured that as long as I was moving the version, I'd update the version # and allow the patcher to run.